src/main/java/htsjdk/variant/vcf/VCFFileReader.java

```diff
@@ -154,7 +177,10 @@ public static IntervalList toIntervalList(final Path path, final boolean include
      *
      * @param file
      * @return
+     *
+     * @deprecated use {@link #toIntervalList} instead
```

Comment:
I think you mean to write `{@link #toIntervalList(Path)}` here; the args are required to get the correct method since `toIntervalList` is overloaded.